Site/ASAB Maestro/Descriptors/seacat-auth.yaml

```diff
@@ -85,5 +85,9 @@ nginx:
       - rewrite ^/auth/api/(.*) /$1 break
       - proxy_pass http://upstream-seacat-auth-3081
 
+    location = /api/cookie-entry:
+      - proxy_pass http://upstream-seacat-auth-3081/cookie/entry
+      - proxy_ignore_headers Cache-Control Expires Set-Cookie
```

Comment:
What is this, please?

Reply:
no clue

Reply:
Ok, so let's remove this endpoint

Reply:
ok, the whole endpoint is needed for the batman introspection. I don't really understand the meaning of removing the headers, but the endpoint is there to make kibana and grafana introspection possible.

Reply:
eh ... is it the landing point?

if yes, then the name should be better

Reply:
Please add the comment:

# This API endpoint is a landing zone for cookie-based authorization, used for Kibana / Grafana